client/src/index.js

```diff
@@ -1,8 +1,247 @@
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+    },
+    button: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(1),
+    },
+  }));
+
+export default function Login() {
+
+    //error slide up
+    const [open, setOpen] = React.useState(false);
+    const [transition, setTransition] = React.useState(undefined);
+
+
+    const state = {
+        email: "",
+        password: "",
+        confirmPassword: "",
+        firstname: "",
+        lastname: "",
+        isLogin: true
+    };
+
+    const updateEmail = e => {
+        state.email = e.target.value
+    }
+    
+    const updatePassword = e => {
+        state.password = e.target.value
+    }
+
+    const updateConfirmPassword = e => {
+        state.confirmPassword = e.target.value
+    }
+
+    const updateFirstname = e => {
+        state.firstname = e.target.value
+    }
+
+    const updateLastname = e => {
+        state.lastname = e.target.value
+    }
+    
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const handleSubmit = () => {
+        //distinguish between login and signup
+        console.log(state.isLogin)
+        //Login
+        if (state.isLogin) {
+            if (document.getElementById("standard-username").value === "" || document.getElementById("standard-password").value === "") {
+                //setTransition();
+                //setOpen(true);
+                console.log("Please fill in all requirements!!!")
+            } else {
+                //login successfully
+            }
+        } else {
+            //Sign up
+            if (document.getElementById("standard-username").value === "" || 
+                document.getElementById("standard-password").value === "" ||
+                document.getElementById("firstname").value === "" ||
+                document.getElementById("lastname").value === "" ||
+                document.getElementById("confirmPassword").value === "") {
+                //setTransition();
+                //setOpen(true);
+                console.log("Please fill in all requirements!!!")
+            } else {
+                //sign up successfully
+            }
+        }
+    }
+
+    const switchToSignupPage = () => {
+        document.getElementById("submitButton").innerHTML = "Sign up"
+        document.getElementById("welcome").innerHTML = "Sign up"
+        document.getElementById("forgotLink").style.display = "none";
+        document.getElementById("createAccountLink").style.display = "none";
+        document.getElementById("backLink").style.display = "block";
+        document.getElementById("confirmTextField").style.display = "block";
+        document.getElementById("firstTextField").style.display = "block";
+        document.getElementById("lastTextField").style.display = "block";
+        state.isLogin = false
+    }
+
+    const switchToLoginPage = () => {
```

Comment:
Same here as above. Reset the state when switching.

Reply:
Fixed